ksqldb-engine/src/test/java/io/confluent/ksql/structured/SchemaKSourceFactoryTest.java

```diff
@@ -263,7 +266,7 @@ public void shouldReplaceOldNonWindowedStreamSourceWithMatchingPseudoColumnVersi
     );
 
     // Then:
-    assertThat(((StreamSource) result.getSourceStep()).getPseudoColumnVersion(), equalTo(OLD_PSEUDOCOLUMN_VERSION));
+    assertThat(((StreamSource) result.getSourceStep()).getPseudoColumnVersion(), equalTo(SystemColumns.LEGACY_PSEUDOCOLUMN_VERSION_NUMBER));
```

Comment:
nit: you can use a static important to shorten appearances of `SystemColumns.LEGACY_PSEUDOCOLUMN_VERSION_NUMBER` to simply `LEGACY_PSEUDOCOLUMN_VERSION_NUMBER`, and similarly for `CURRENT_PSEUDOCOLUMN_VERSION_NUMBER`.